File: task2.go
package main

import "fmt"

func main() {
	fmt.Println(convertNumber("5"))
	fmt.Println(convertNumber("25"))
	fmt.Println(convertNumber("125"))
}

func convertNumber(number string) string {
	switch len(number) {
	case 1:
		number = "000" + number
	case 2:
		number = "00" + number
	case 3:
		number = "0" + number
	}

	return number
}

File: Task2.php
<?php

function convertNumber($number) {
    $numberStringResult = '';
    if (strlen($number) >= 1 && strlen($number) <= 3) {
        for ($i = 1; $i <= (4-strlen($number)); $i++){
            $numberStringResult .= '0';
        }
        $numberStringResult .= $number;
    }

    return $numberStringResult;
}

convertNumber(5);
?>